python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -475,16 +475,21 @@ async def check_comp_heartbeat(self, component):
 
         Raises
         ------
-        asyncio.TimeoutError
+        RuntimeError
             If the component does not send heartbeats in `self.fast_timeout`
             seconds.
 
         """
 
         while True:
-            await getattr(self.rem, component).evt_heartbeat.next(
-                flush=True, timeout=self.fast_timeout
-            )
+            try:
+                await getattr(self.rem, component).evt_heartbeat.next(
+                    flush=True, timeout=self.fast_timeout
+                )
+            except asyncio.TimeoutError:
+                raise RuntimeError(
+                    f"No heartbeat from {component} received in {self.fast_timeout}s."
+                )
```

Comment:
Why this change? It seems unrelated to the ticket and seems unnecessary -- TimeoutError is the natural exception to raise. If you merely want to add a message then you can raise a new instance of asyncio.TimeoutError.

Reply:
The original exception doesn't say which component missed the heartbeat, so it is hard to identify the source of the problem. I will re-raise `asyncio.TimeoutError`.

Reply:
Sounds good. Thank you for the explanation.